lib/routes/account.js

```diff
@@ -419,7 +421,7 @@ module.exports = function (
                 //  * the request wants keys, since unverified sessions are fine to use for e.g. oauth login.
                 //  * the email is verified, since content-server triggers a resend of the verification
                 //    email on unverified accounts, which doubles as sign-in confirmation.
-                if (! requestHelper.shouldEnableTokenVerification(emailRecord, config, request)) {
+                if (! features.isSigninConfirmationEnabled(emailRecord.uid, emailRecord.email, request)) {
```

Comment:
I really like having these factored out into an explicit `features` module, it feels right.